lang/tr/README.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: page
+title: archipelago 🐚 🌊 🐬 🏄 🌴 🥥 🩴 🏖️ 👙
+permalink: /
+---
+
+[English](README.md)
+[Spanish](lang/es/README.md)
+
```

Comment:
Missing language button for Turkish

Reply:
I thought we should add the Turkish button to other languages. Are you sure that you want me to add it?